scripts/projectView.js

```diff
@@ -0,0 +1,91 @@
+// Configure a view object, to hold all our functions for dynamic updates and article-related event handlers.
+var portfolioPage = {};
+
+portfolioPage.populateFilter = function(template_id) {
+  var $source = $(template_id).html();
+  var filter = Handlebars.compile($source);
+  return filter({myProjects: projects});
+};
+
+
+Handlebars.registerHelper('list', function(items, options) {
```

Comment:
I don't think this is working how you want it to and I'm not actually sure what you're trying to do here.